enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/ConfiguredAuthScenariosInteractionTestBase.java

```diff
@@ -80,4 +97,60 @@ public void shouldAllowRoleCallCreateNewTokensProceduresWhenConfigured() throws
         assertEmpty( noneSubject, "CALL db.createRelationshipType('MySpecialRelationship')" );
         assertEmpty( noneSubject, "CALL db.createProperty('MySpecialProperty')" );
     }
+
+    @Test
+    public void shouldWarnWhenUsingNativeAndOtherProvider() throws Throwable
+    {
+        configuredSetup( stringMap( SecuritySettings.auth_providers.name(), "native ,LDAP" ) );
+        assertSuccess( adminSubject, "CALL dbms.security.listUsers", r -> assertKeyIsMap( r, "username", "roles", userList ) );
+        GraphDatabaseFacade localGraph = neo.getLocalGraph();
+        InternalTransaction transaction = localGraph
+                .beginTransaction( KernelTransaction.Type.explicit, StandardEnterpriseSecurityContext.AUTH_DISABLED );
+        Result result =
+                localGraph.execute( transaction, "EXPLAIN CALL dbms.security.listUsers", Collections.emptyMap() );
+        assertThat(
+                containsNotification( result, "Native user management procedures will not affect non-native users." ),
+                equalTo( true ) );
+        transaction.success();
+        transaction.close();
+    }
+
+    @Test
+    public void shouldNotWarnWhenUsingNativeAndOtherProvider() throws Throwable
```

Comment:
And what about a test for no warnings when we have no native configured (eg. LDAP together with plugins)